airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/sponsored_display.py

```diff
@@ -12,11 +12,17 @@ class SponsoredDisplayCampaigns(SubProfilesStream):
     https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#/Campaigns
     """
 
+    def __init__(self, *args, **kvargs):
```

Comment:
Function argument `**kwargs` is written with `w`. See https://book.pythontips.com/en/latest/args_and_kwargs.html#usage-of-kwargs. Since you're editing this file, please fix the naming across the whole file.